Engineering/Templates/Terraform/VSCode/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "azurerm"{
+    version = "~>1.5"
+}
+terraform{
+    backend "azurerm" {
+    }
+}
+
+locals {
+    location_suffixes = {
+        centralus       = "cus"
```

Comment:
Which plugin/extension/??? is used to format the spacing here and make the `=` sign line up? It arguably looks neater, but unless the developer has whatever automatically did this (hopefully not manually), it won't look aligned without manual effort.

My suggestion would be to remove the alignment auto-formatting here for the template. (sorry if this is a bit of a nit)

Reply:
Hey @davidkpiano, i have updated the .tf files to not have the alignment indicated. To better resolve your point about "which plugin/extension/???", i have added an extensions.json that will cause vs code to recommend the extension favored for terraform development. I believe this should resolve this issue.